templates/goods_list.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1>Goods to buy</h1>
+
+<div>
+    {% for good in page_obj %}
+        <div>
+            <b>{{ good.title }}</b><br>
+            {% if good.image %}<img src="{{good.image.url}}">{% endif %}<br>
+            {{ good.description }}<br>
+            {{ good.price }} USD<br>
+            <form method="post" action="{% url 'goods' %}">
+                {% csrf_token %}
+                <input type="hidden" name="pk" value="{{good.pk}}">
+                <input type="number" value="1" name="quantity" min="1" max="{{good.in_stock}}">
+                <input type="hidden" name="price" value="{{good.price}}">
```

Comment:
what for pk and price?? it's the same instance

Reply:
I added it for use in the refund "frozen" price - which won't be dependent on any changes 
now it is deleted